config/default.json

```diff
@@ -0,0 +1,4 @@
+{
+    "mongoURI": "mongodb+srv://admin:admin@cluster0.vhd1w.mongodb.net/test?retryWrites=true&w=majority",
```

Comment:
I'm hesitant to place these links directly into the repo as they are very specific endpoints for mongodb and for our auth hashing -- thinking we should gitignore them for security purposes and just have everyone manually create this file as necessary